# Transcriptional analysis of ASCL1 and the effects of neurogenesis in GABAergic Neurons
 
![Powerpoint  Transcriptional analysis of ASCL1 and the effects of neurogenesis in GABAergic Neurons_page-0001](https://github.com/gappeah/Transcriptional-analysis-of-ASCL1-and-the-effects-of-neurogenesis-in-GABAergic-Neurons/assets/114095068/e623ec3e-0128-4d8b-9cf5-39338cb0a906)


My final year project which I completed in March 2021. 

## Aims
* To understand the role of regulation of ASCL1 within GABAergic neurons
* To Identify co-expression groups that occur alongside ASCL1
* To investigate the role of ASCL1 targets genes by gene ontology

These aims will be investigated by collecting the next-generation sequences and performing a comparative analysis on existing gene data sets to identify expressions of ASCL1 target genes in different cell clusters
achieved by using bioinformatics software. This will result in the 2D graphical representation of the genes and their expression patterns.

## Method
Leveraged bioinformatics platforms, Galaxy and GEO2R and processed 85,744 entries across five datasets to identify differentially expressed genes. 
Utilised data visualisation tools to highlight overlapping genes and to visualise protein interaction networks. Generated volcano plots and performed data modelling using PCA and UMAP to illustrate sample similarities and data variability.

## Results and Discussions
1. GAD2 and CALB1/2 are associated with synaptic communication.
2. GATA2 is associated with GABAergic differentiation, NOTCH2 inhibition promotes neural genes related to neurogenesis and PITX2's role appears to be confined in neurogenesis in the midbrain. 
3. Slit2 and its role remains unclear in the process of neurogenesis and may play a minimal role in neural differentiation and maintaining axonal morphology.


## Authors

- [@gappeah](https://github.com/gappeah)


## References
* [Galaxy](https://galaxyproject.org/) is an open, web-based platform for data-intensive biomedical research. It allows users to perform accessible, reproducible, and transparent computational research.
* [GEO2R](https://www.ncbi.nlm.nih.gov/geo/geo2r/) is an interactive web tool that allows users to compare two or more groups of samples in the Gene Expression Omnibus (GEO) database to identify differentially expressed genes.
* [g:Profiler](https://biit.cs.ut.ee/gprofiler/) is a web server for functional enrichment analysis and conversions of gene lists, identifiers, and various other information related to gene function and pathways.
* [STRING](https://string-db.org/) is a database of known and predicted protein-protein interactions. It includes direct (physical) and indirect (functional) associations.
* [InteractiVenn](http://www.interactivenn.net/) is a web-based tool for the visualization of Venn diagrams for multiple sets with a user-friendly interface.
* [ArrayExpress](https://www.ebi.ac.uk/arrayexpress/) is a public database of gene expression data and a repository for high-throughput functional genomics data.
* [Gene Expression Omnibus (GEO)](https://www.ncbi.nlm.nih.gov/geo/) is a public repository that archives and freely distributes comprehensive sets of gene expression data.



